crates/sui-core/src/execution_cache.rs

```diff
@@ -131,13 +131,11 @@ pub fn choose_execution_cache(config: &ExecutionCacheConfig) -> ExecutionCacheCo
         }
     }
 
-    if std::env::var(DISABLE_WRITEBACK_CACHE_ENV_VAR).is_ok()
-        || matches!(config, ExecutionCacheConfig::PassthroughCache)
-    {
-        ExecutionCacheConfigType::PassthroughCache
-    } else {
-        ExecutionCacheConfigType::WritebackCache
+    if std::env::var(DISABLE_WRITEBACK_CACHE_ENV_VAR).is_ok() {
+        error!("DISABLE_WRITEBACK_CACHE is no longer respected. WritebackCache is the default.");
```

Comment:
should this be fatal?

Reply:
i don't think so? i don't think this will affect anyone anyway.